test/systemtest/earlgrey/test_sim_verilator.py

```diff
@@ -217,24 +225,21 @@ def app_silicon_creator_selfchecking(request, bin_dir):
 
     app_config = request.param
 
-    if 'name' not in app_config:
+    if not all(key in app_config for key in ('name', 'signing_key')):
```

Comment:
We really should de-specialize this function again and just make the bootrom a variant in the app_selfchecking() function (and also use the same config -- no need for two config files, it's just a matrix test). 

Either way, that's for another day (if you're bored waiting for a simulation, or so)

Reply:
I agree, we should revisit this when we add additional tests, e.g. for different OTP images.

Reply:
Created #6669.

Reply:
Thanks @alphan!